doc/modules/fuzzy_check.md

```diff
@@ -11,39 +11,122 @@ is responsible for teaching fuzzy storage with message patterns.
 ## Fuzzy patterns
 
 Rspamd uses the `shingles` algorithm to perform a fuzzy match of messages. This algorithm
-is probabilistic and uses word chains to detect some common patterns, and thus filter
+is probabilistic and uses word chains to detect some common spam patterns, and thus filter
```

Comment:
I am not agree with this change. IMO the `common patterns` phrase is related to the shingles algorithm in general. Even in case of Rspamd it is used to detect `ham` patterns as well (FUZZY_WHITE).

Reply:
Why else do we use rspamd? ;)